CHANGELOG.md

```diff
@@ -1,6 +1,93 @@
 Changelog
 =========
 
+Version 0.10.0 (October 3, 2021)
+--------------------------------
+
+Highlights of this release:
+
+- A new `sjoin_nearest()` function to join based on proximity, with the
+  ability to set a maximum search radius (#1865). In addition, the `sindex`
+  attribute gained a new method for a "nearest" spatial index query (#1865,
+  #2053).
+- A new `explore()` method on GeoDataFrame and GeoSeries with native support
+  for interactive visualization based on folium / leaflet.js (#1953)
+- The `geopandas.sjoin()`/`overlay()`/`clip()` functions are now also
+  available as methods on the GeoDataFrame (#2141, #1984, #2150).
+
+New features and improvements:
+
+- Add support for pandas' `value_counts()` method for geometry dtype (#2047).
+- The `explode()` method has a new `ignore_index` keyword (consistent with
+  pandas' explode method) to reset the index in the result, and a new
+  `index_parts` keywords to control whether a cumulative count indexing the
+  parts of the exploded multi-geometries should be added (#1871).
+- `points_from_xy()` is now available as a GeoSeries method (#1936).
```

Comment:
```suggestion
- `points_from_xy()` is now available as a GeoSeries method `from_xy` (#1936).
```

Just a minor change to make it explicit that the method is not `GeoSeries.points_from_xy` but `GeoSeries.from_xy`.

Reply:
Hmm, I didn't actually realize that in the PR that added this. I understand that the shorter method name might be a bit nicer, but the inconsistent naming between the function and the method is also a bit unfortunate ...

Reply:
I think it was actually my proposal, to keep it consistent with the `GeoSeries.from_*` bunch of methods.

Reply:
Yes, there is also the consistency with that :)